src/h3/api/basic_int/__init__.py

```diff
@@ -596,13 +603,6 @@ def h3shape_to_cells_experimental(h3shape, res, flags=0):
     return _out_collection(mv)
 
 
-def polygon_to_cells_experimental(h3shape, res, flags=0):
```

Comment:
This is the name of the function in the C library and on the docs site. Why is the alias being removed?

Reply:
I think we discussed this previously, and the preferred function name in `h3-py` is `h3shape_to_cells` since `polygon` doesn't account for multipolygons. We added the alias `polygon_to_cells()` in `h3-py` just as a pointer for folks to discover the preferred function.

As this function is experimental, and we already have the pointer, I think adding an additional alias is redundant. Also, we'll get back to the status quo when we merge the experimental function into the primary function.